content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -44,14 +44,11 @@ external IP address.
 
 1. Run a Hello World application in your cluster:
 
-{{< codenew file="service/load-balancer-example.yaml" >}}
-
-```shell
-kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
-```
-
-
-The preceding command creates a
+   {{< codenew file="service/load-balancer-example.yaml" >}}
+   ```shell
+   kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
+   ```
+    The preceding command creates a
```

Comment:
can you help unindent line 51-57 by one space?

Reply:
lines 140~144 and 114~117 are indent with four spaces.

Reply:
Do NOT indent by 4 spaces. The markdown engine sometimes mistreat such lines as code blocks.

Reply:
fixed